src/main.rs

```diff
@@ -441,40 +439,41 @@ fn parse_numeric_range_opts(
     min_value: u8,
     max_value: u8,
 ) -> Result<HashSet<u8>, String> {
-    let one_item = Regex::new(format!(r"^[{}-{}]$", min_value, max_value).as_ref()).unwrap();
-    let multiple_items = Regex::new(
-        format!(
-            r"^([{}-{}])(,|-)([{}-{}])$",
-            min_value, max_value, min_value, max_value
-        ).as_ref(),
-    ).unwrap();
+    let error_message = "Not a valid input";
     let mut items = HashSet::new();
 
-    if one_item.is_match(input) {
-        items.insert(input.parse::<u8>().unwrap());
-        return Ok(items);
+    // one value
+    if let Some(one_value) = input.parse::<u8>().ok() {
```

Comment:
You can write this more efficiently as `if let Ok(one_value) = input.parse::<u8>()`